.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,88 @@
+name: Pull Request - Java Build & Lint & beta release
+
+on: [pull_request]
```

Comment:
Need to set up a separate workflow for pushes to master (but they won't be beta/prerelease)